CHANGELOG.md

```diff
@@ -4,6 +4,47 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.15](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.15) - 2021-08-TBD
+
+Zebra's latest alpha contains the last of the changes to zebra chain state ahead of NU5 testnet activation and before concentrating on mempool work. The remaining NU5 validation will be completed prior to NU5 mainnet activation.
+
+### Added
+
+#### Network Upgrade 5
+
+- ZIP-209: Prohibit Negative Shielded Chain Value Pool Balances Implementation (#2546, #2554, #2561, #2566, #2569, #2576, #2577, #2578, #2596)
```

Comment:
The chain value balance checks are still incomplete, can we say that here?

Also we might want to split out "Check remaining transaction value is non-negative".
That is a separate consensus rule that was completed by this PR:
https://github.com/ZcashFoundation/zebra/pull/2566

Reply:
done in f467eae